spec/ngsild_csource_schema.json

```diff
@@ -0,0 +1,137 @@
+{
+    "definitions": {
+        "CsourceRegistrationParams": {
+            "type": "object",
+            "properties": {
+                "timestamp": {
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "$ref": "ngsild_schema.json#/definitions/TimeInterval"
+                    }
+                },
+                "location": {
+                    "$ref": "http://json.schemastore.org/geojson#/definitions/geometry"
+                },
+                "expires": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "description": {
+                    "type": "string"
+                }
+            }
+        },
+        "CsourceRegistrationMandatoryParams": {
```

Comment:
do we really need to make this separation?

Reply:
Yes from my point of view, they are not mixed with CsourceRegistrationParams since CsourceRegistrationParams are used separetely by CsourceRegistrationFragment, where CsourceRegistrationFragment contains mandatory params without the 'required' restriction since are option for fragments.